server/views/dashboard/index.njk

```diff
@@ -0,0 +1,50 @@
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName %}
+{% set mainClasses = "app-container govuk-body" %}
+{% set bodyClasses = "dashboard--index" %}
+
+{% block content %}
+
+  {% include "../_messages.njk" %}
+
+  <h1>Approved Premises</h1>
```

Comment:
I think it's better to pass the title in from the controller

Reply:
Good call 👍